### ISSUE TITLE
Module load error jina.hub.encoders.nlp.CLIPTextEncoder

### ISSUE THREAD

**Issue by huanbd** [jina-ai/jina-hub#5785]
Hi, I'm follow [this guide ](https://docs.jina.ai/chapters/my_first_jina_app/), but when run `python app.py -t index` an error occurs
```
/jina_env/lib/python3.7/site-packages/jina/importer.py:267: UserWarning: due to the missing dependencies or bad implementations, ['jina.hub.encoders.nlp.CLIPTextEncoder'] can not be imported if you are using these executors/drivers, they wont work. You can use `jina check` to list all executors and drivers
  f'due to the missing dependencies or bad implementations, '
Traceback (most recent call last):
  File "app.py", line 62, in <module>
    raise NotImplementedError(f'unsupported mode {sys.argv[1]}')
NotImplementedError: unsupported mode -t
```
When I run `jina check`
```
 JINA@24262[I]:
AVAILABLE HUB EXECUTORS


Load  Class                     Module                                   Dependency
----- ------------------------- ---------------------------------------- ----------
✗ Module load error jina.hub.encoders.nlp.CLIPTextEncoder    No module named 'clip'
✓ AlbumentationsCrafter     jina.hub.crafters.image.AlbumentationsCrafter JAMLCompatible▸BaseExecutor▸BaseCrafter▸AlbumentationsCrafter
```

My env is Python3.7, CentOS 7

**Comment by JoanFM** [comment]
Hey @huanbd ,

this is not a big problem, is just a warning about an executor not being available, but it does not affect the functionality of the rest, however we should avoid this warning, and thus we are creating this PR #5786

**Comment by huanbd** [comment]
Hi @JoanFM ,

Thanks for your fast response, but how can I run this guide [https://docs.jina.ai/chapters/my_first_jina_app/](https://docs.jina.ai/chapters/my_first_jina_app/)

Thank you,

**Comment by JoanFM** [comment]
> Hi @JoanFM ,
> 
> Thanks for your fast response, but how can I run this guide https://docs.jina.ai/chapters/my_first_jina_app/
> 
> Thank you,

U can still run the guide ignoring this warning

**Comment by huanbd** [comment]
When I run to [this step](https://docs.jina.ai/chapters/my_first_jina_app/#index-flow) `python app.py -t index` error occurs
```
/jina_env/lib/python3.7/site-packages/jina/importer.py:267: UserWarning: due to the missing dependencies or bad implementations, ['jina.hub.encoders.nlp.CLIPTextEncoder'] can not be imported if you are using these executors/drivers, they wont work. You can use `jina check` to list all executors and drivers
  f'due to the missing dependencies or bad implementations, '
Traceback (most recent call last):
  File "app.py", line 62, in <module>
    raise NotImplementedError(f'unsupported mode {sys.argv[1]}')
NotImplementedError: unsupported mode -t
```
;( , follow this guide  `python app.py -t index` contain` -t`, but in `app.py` not implement that, I remove -t , It's run. 
Waste your time :(

**Comment by JoanFM** [comment]
> When I run to [this step](https://docs.jina.ai/chapters/my_first_jina_app/#index-flow) `python app.py -t index` error occurs
> 
> ```
> /jina_env/lib/python3.7/site-packages/jina/importer.py:267: UserWarning: due to the missing dependencies or bad implementations, ['jina.hub.encoders.nlp.CLIPTextEncoder'] can not be imported if you are using these executors/drivers, they wont work. You can use `jina check` to list all executors and drivers
>   f'due to the missing dependencies or bad implementations, '
> Traceback (most recent call last):
>   File "app.py", line 62, in <module>
>     raise NotImplementedError(f'unsupported mode {sys.argv[1]}')
> NotImplementedError: unsupported mode -t
> ```
> 
> ;( , follow this guide `python app.py -t index` contain` -t`, but in `app.py` not implement that, I remove -t , It's run.
> Waste your time :(

Ok I see

**Comment by JoanFM** [comment]
Closed by #5786 

Btw, I opened a ticket in our main repo to keep track of the underlying problem, but closing the issue related to CLIPTextEncoder
